controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -327,3 +344,82 @@ func (r *OCSInitializationReconciler) getEnableNFSKeyValue() string {
 
 	return "false"
 }
+
+func (r *OCSInitializationReconciler) reconcileUXBackendSecret(initialData *ocsv1.OCSInitialization) error {
+
+	var err error
+
+	secret := &corev1.Secret{}
+	secret.Name = "ux-backend-proxy"
+	secret.Namespace = initialData.Namespace
+
+	b := make([]byte, 20)
+	_, err = rand.Read(b)
+	if err != nil {
+		r.Log.Error(err, "Failed to generate random string for ux-backend secret")
+		return err
+	}
+	session_secret := base64.URLEncoding.EncodeToString(b)[:20]
+
+	secret.StringData = map[string]string{"session_secret": session_secret}
```

Comment:
minor: session_secret can be inlined

Reply:
not needed, made it multiline as ohad requested